### ISSUE TITLE
A question about tokenizer file

### ISSUE THREAD

**Issue by hideinhat** [win5923/TrOCR-Handwritten-Mathematical-Expression-Recognition#2]
Hello, I am trying to finetune the TrOCR to recognize latex math expression on my custom dataset.
Do I need to train a custom tokenizer for latex format?
Or is the pretrained tokenizer one included with TrOCR good enough?
Any pointers or help would be greatly appreciated.
Thank you!

**Comment by win5923** [comment]
Yes, my results using the pre-trained tokenizer were not satisfactory. I believe it's necessary to train a dedicated tokenizer based on mathematical symbols. Perhaps there already exists.

```
processor = TrOCRProcessor.from_pretrained("microsoft/trocr-base-handwritten")
```

**Comment by hideinhat** [comment]
> Yes, my results using the pre-trained tokenizer were not satisfactory. I believe it's necessary to train a dedicated tokenizer based on mathematical symbols. Perhaps there already exists.
> 
> ```
> processor = TrOCRProcessor.from_pretrained("microsoft/trocr-base-handwritten")
> 
Thank you for reply. I find one on https://huggingface.co/witiko/mathberta. I will try on it.
